setup.py

```diff
@@ -34,7 +34,7 @@ def read_text_file(fname):
 
 setup(
     name="sqs-log-handler",
-    version="1.1.0",
+    version="1.1.1+instructure",
```

Comment:
I did not see this kind of versioning scheme before. Is this intended?

Reply:
It was a mistake that this got added to the PR. I will re-base and re-submit.